extensions/wikia/AdEngine/js/spec/slot/scrollHandler.spec.js

```diff
@@ -57,6 +57,10 @@ describe('ext.wikia.adEngine.slot.scrollHandler', function () {
 		shouldNotBeRefreshed({reloadedViewMax: 0});
 	});
 
+	it('Prefooter should not be refreshed when trigger is not scroll', function () {
+		shouldNotBeRefreshed({reloadedViewMax: 0, trigger: 'delay.5s'});
```

Comment:
`reloadedViewMax` can't be `0` because slot would not be refreshed due to this value as well.
Please, change it to `-1` or better I would like to see: `shouldNotBeRefreshed({trigger: 'delay.5s'});`